### PR TITLE
[ios] Add support of website:menu tag to ios

### PR DESCRIPTION
**Pull request by kirylkaveryn**
This PR continues https://github.com/organicmaps/organicmaps/pull/8052

@biodranik where should we add the tag into the `editor.config`? Because now its not visible in UI. 
Tag is not currently used anywhere.

https://github.com/organicmaps/organicmaps/blob/4ecf03a008ec4f025e41b8302fd48ac918cf4b83/data/editor.config#L75-L77

Simulator Screen Recording - iPhone 15 Pro - 2024-05-21 at 22 05 52

Here is The Edit screen with the menu section only to show how it would be in UI:
Simulator Screen Recording - iPhone 15 Pro - 2024-05-12 at 17 27 39

### CODE REVIEW

**Review by biodranik — APPROVED** [review]
Дзякуй!

File: iphone/Maps/UI/PlacePage/Components/PlacePageInfoViewController.swift [hunk 1]
```diff
@@ -158,7 +160,20 @@ class PlacePageInfoViewController: UIViewController {
         self?.delegate?.didCopy(website)
       })
     }
-    
+
+    if let websiteMenu = placePageInfoData.websiteMenu {
+      // Strip website url only when the value is displayed, to avoid issues when it's opened or edited.
```
[the remark this answers is not in the document]
Comment (kirylkaveryn) [reply iphone/Maps/UI/PlacePage/Components/PlacePageInfoViewController.swift:165]: Fixed!

File: iphone/CoreApi/CoreApi/PlacePageData/Common/PlacePageInfoData.mm [hunk 2]
```diff
@@ -83,7 +83,7 @@ - (instancetype)initWithRawData:(Info const &)rawData ohLocalization:(id<IOpenin
           if (value == "yes")
             _driveThrough = NSLocalizedString(@"drive_through", nil);
           break;
-
+        case MetadataID::FMD_WEBSITE_MENU: _websiteMenu = ToNSString(value); break;
         default:
```
[the remark this answers is not in the document]
Comment (kirylkaveryn) [reply iphone/CoreApi/CoreApi/PlacePageData/Common/PlacePageInfoData.mm:87]: Good point!

File: iphone/CoreApi/CoreApi/PlacePageData/Common/PlacePageInfoData.mm [hunk 3]
```diff
@@ -83,9 +83,34 @@ - (instancetype)initWithRawData:(Info const &)rawData ohLocalization:(id<IOpenin
...
+        case MetadataID::FMD_RATINGS: break;
+        case MetadataID::FMD_AIRPORT_IATA: break;
+        case MetadataID::FMD_BRAND: break;
+        case MetadataID::FMD_DURATION: break;
+        case MetadataID::FMD_DESTINATION: break;
+        case MetadataID::FMD_DESTINATION_REF: break;
+        case MetadataID::FMD_JUNCTION_REF: break;
+        case MetadataID::FMD_BUILDING_MIN_LEVEL: break;
+        case MetadataID::FMD_LOCAL_REF: break;
+        case MetadataID::FMD_COUNT: break;
```
Comment (vng) [inline iphone/CoreApi/CoreApi/PlacePageData/Common/PlacePageInfoData.mm:113]: What for these cases? They don't bring any value comparing to default. Paranoid ObjC warning?

Comment (kirylkaveryn) [reply iphone/CoreApi/CoreApi/PlacePageData/Common/PlacePageInfoData.mm:113]: Yes it's only for warnings 

https://github.com/organicmaps/organicmaps/pull/8244#discussion_r1608849200


I agree that empty 27cases seems awkward

Comment (vng) [reply iphone/CoreApi/CoreApi/PlacePageData/Common/PlacePageInfoData.mm:113]: Please, remove it. This warning will not help you whether it is present or not.

Comment (kirylkaveryn) [reply iphone/CoreApi/CoreApi/PlacePageData/Common/PlacePageInfoData.mm]: Fixed! Thanks!

**Pull request merged**